modules/server/src/network/aggregations/index.ts

```diff
@@ -44,21 +44,46 @@ export const resolveToNetworkAggregation = (
  * Takes an array of the same aggregation type and computes the singular type
  * eg. NumericAggregation => NetworkNumericAggregation
  *
+ * Note for operations on Buckets -  size can be large
```

Comment:
"size" alone is ambiguous, in that a large bucket doesn't add complexity, whereas the count of buckets is what I think you're trying to highlight here
```suggestion
 * Note for operations on Buckets -  the size of the array can be large (e.g. total bucket count), complicating lookups, etc.
```